torchaudio/datasets/voxceleb1.py

```diff
@@ -0,0 +1,82 @@
+import os
+from pathlib import Path
+from typing import Tuple, Union, Optional
+
+import torchaudio
+from torch import Tensor
+from torch.hub import download_url_to_file
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import (
+    extract_archive,
+)
+
+
+_RELEASE_CONFIGS = {
+    "dev": {
+        "url": "https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_dev_wav.zip",
+        "checksum": "ff3b4ce606718a2d221299d21f1dded47097907762c6783e47fe823cad9f001e",
+    },
+    "test": {
+        "url": "https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_test_wav.zip",
+        "checksum": "8de57f347fe22b2c24526e9f444f689ecf5096fc2a92018cf420ff6b5b15eaea",
+    },
+}
+
+
+class VoxCeleb1(Dataset):
+    """Create VoxCeleb1 Dataset.
+
+    Args:
+        root (str or Path): Path to the directory where the dataset is found or downloaded.
+        subset (str or None, optional): Subset of the dataset to use. Options: ["dev", "test"]. (Default: ``None``)
```

Comment:
Can you mention the default behavior if `None`? Also with the current implementation, `None` will throw an error because of `_RELEASE_CONFIGS[subset]["url"]` accessing an invalid key